src/OrchardCore.Modules/OrchardCore.Alias/Migrations.cs

```diff
@@ -22,20 +22,19 @@ public int Create()
                 .Attachable()
                 .WithDescription("Provides a way to define custom aliases for content items."));
 
-            // NOTE: The Alias Length has been upgraded from 64 characters to 767.
-            // For existing SQL databases update the AliasPartIndex tables Alias column length manually.
-            // INFO: The Alias Length is now of 735 chars, but this is only used on a new installation.
```

Comment:
As said in https://github.com/OrchardCMS/OrchardCore/pull/14491#issuecomment-1760236420

> ... `DocumentId (26)` is wrong as it is at most a bigint (size = 8 bytes = `2 chars size`), the 2 booleans may not share the same bytes but `one char size` (4 bytes) is sufficient, anyway we kept a margin of 4 bytes.

Not so important to assume one "char" per boolean, but at least comment `DocumentId (2)` which here will allow `Alias(738)`, and if we want to keep a margin of 4 bytes => `Alias(734)`.

Then the following comments could be removed, and maybe revert back `MaxAliasLength` to its original value, from memory it was 1024, but maybe too much, I don't know, just for info.

https://github.com/OrchardCMS/OrchardCore/blob/f377a07462ea7843094c6872c12c43c74727f728/src/OrchardCore.Modules/OrchardCore.Alias/Models/AliasPart.cs#L7-L13

Same remark for other indexes.

Reply:
> Awesome. I also see that the Text field is limited to 766, was it done this way because of indexing issues too? If so we could update it, at least for new installs.

Not sure why 766. But most likely related to the same limit. i don't know if we should increase it. I mean we could, but would would be a good length? 1024? The longer the column, may require more disk space from the server to maintained it. If you have a magic number, then I can update this and the Alias to use the same.

Reply:
@jtkech thanks for pointing out the seize of `DocumentId` I know it was long but assumed it was a char :). Anyway, I update it.